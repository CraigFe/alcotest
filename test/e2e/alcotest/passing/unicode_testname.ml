let () =
  Alcotest.run "suite-name"
    [
      ( "ðŸ”¥",
        [
          Alcotest.test_case "Non ASCII unicode character" `Quick (fun () -> ());
        ] );
      ( "ðŸ”¥a-b",
        [
          Alcotest.test_case "Non ASCII and ASCII characters" `Quick (fun () ->
              ());
        ] );
    ]
