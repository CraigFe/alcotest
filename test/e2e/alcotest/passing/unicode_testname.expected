Testing `suite-name'.
This run has ID `<uuid>'.

  ...           U+1F525             0   Non ASCII unicode character.  [OK]          U+1F525             0   Non ASCII unicode character.
  ...           U+1F525a-b          0   Non ASCII and ASCII characters.  [OK]          U+1F525a-b          0   Non ASCII and ASCII characters.

Full test results in `<build-context>/_build/_tests/suite-name'.
Test Successful in <test-duration>s. 2 tests run.
